lib/caasp.pm

```diff
@@ -33,6 +33,11 @@ sub process_reboot {
     script_run("reboot", 0) if $trigger;
 
     reset_consoles;
+    if (check_var('VIRSH_VMM_FAMILY', 'xen')) {
+        sleep 7;
+        select_console 'sut';
+    }
```

Comment:
Well, it's not implemented in wait_boot and every test module handles this itself for now
 # grep -ri "select_console.*sut" found 7 examples